client/extensions/woocommerce/state/ui/shipping/zones/methods/utils.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+export const getMethodName = ( methodType ) => {
+	switch ( methodType ) {
+		case 'flat_rate':
+			return translate( 'Flat Rate' );
+		case 'free_shipping':
+			return translate( 'Free Shipping' );
+		case 'local_pickup':
+			return translate( 'Local Pickup' );
+		default:
+			return translate( 'Unknown shipping method' );
+	}
+};
+
+export const getMethodSummary = ( method ) => {
+	switch ( method.methodType ) {
+		case 'flat_rate':
+			return translate( 'Cost: %s', { args: [ method.cost ] } );
+		case 'free_shipping':
+			if ( ! method.requires ) {
+				return translate( 'For everyone' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **35** times:
`translate( 'Everyone', { context: 'Filter label for posts list'} )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/14275-shipping-methods-redux)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).